fda/grid.py

```diff
@@ -44,6 +45,242 @@ def _list_of_arrays(original_array):
     return list(new_array)
 
 
+class _GridInterpolator:
```

Comment:
I think this class is to big to stay in this module. Maybe it should be moved to another (maybe private?) one.

Reply:
Changed in 8c58331.